cni-plugin/pkg/upgrade/migrate.go

```diff
@@ -63,9 +63,9 @@ func Migrate(ctxt context.Context, c client.Interface, nodename string) error {
 
 	// Check to see if the system is still using host-local
 	// by checking the existence of the path.
-	log.Info("checking host-local IPAM data dir dir existence...")
+	log.Info("checking host-local IPAM data dir existence...")
```

Comment:
Sometimes word duplications are very intentional and necessary (there were a couple of `that that`s that were appropriate), sometimes they're definitely wrong, here I'm only relatively confident.